src/traces/bar/plot.js

```diff
@@ -530,5 +519,32 @@ function calcTextinfo(calcTrace, index, xa, ya) {
         if(hasFlag('final')) text.push(formatNumber(final));
     }
 
+    if(trace.type === 'funnel') {
+        if(hasFlag('value')) text.push(formatNumber(cdi.s));
```

Comment:
Do we have a mock testing `arrayOk` `textinfo` settings?

Reply:
No we don't have any. 
Should `textinfo` be `arrayOk`?
I think in `pie` it is not. Anyway for the moment `arrayOk`: `false` is added here: https://github.com/plotly/plotly.js/pull/3817/commits/8d02bcff0f997dd901a89d9b08af1199ef705cbb for better clarification.

Reply:
I can see value in having `arrayOk:true` for `textinfo`, but no need to do this in this PR.